catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/resource/download/ReliableResourceDownloaderTest.java

```diff
@@ -61,6 +65,8 @@
 import ddf.catalog.resource.ResourceNotSupportedException;
 import ddf.catalog.resourceretriever.ResourceRetriever;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
Now that we've added PowerMock, can you make sure that we're still getting coverage for `ReliableResourceDownloader`? PowerMock sometimes affects JaCoCo and prevents it from seeing the lines that are actually covered by the unit test.

If that's the case, you can take a look at https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-federationstrategy/pom.xml and https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-federationstrategy/src/test/java/ddf/catalog/federation/impl/FederationStrategyTest.java to see how to configure PowerMock to use `@Rule` instead of `@RunWith(PowerMockRunner.class)`.